apps/api-extractor/src/api/CompilerState.ts

```diff
@@ -83,7 +82,7 @@ export class CompilerState {
     const program: ts.Program = ts.createProgram(analysisFilePaths, commandLine.options, compilerHost);
 
     if (commandLine.errors.length > 0) {
-      const errorText: string = TypeScriptMessageFormatter.format(commandLine.errors[0].messageText);
+      const errorText: string = `${commandLine.errors[0].messageText};`;
```

Comment:
We can remove the trailing semicolon.  `formattedErrors.join('; ');` wouldn't have added any delimiter for just one item.

Reply:
Done in https://github.com/microsoft/rushstack/pull/1615/commits/e70461520e16cc4031a9a9e26853f97fde4b88bc.